src/partials/footer.html

```diff
@@ -0,0 +1,29 @@
+<footer class="footer">
+  <div class="footer__container">
+    <div class="footer__logo">
+      <svg class="footer__icon">
+        <use href="./images/sprite.svg#icon-logo"></use>
+      </svg>
+      <div class="footer__tablet">
```

Comment:
Podobnie, nazwa "footer__tablet" wydaje mi się myląca. Wszystkie te elementy są widoczne we wszystkich wielkościach strony.